static/app/views/dashboardsV2/releasesSelectControl.tsx

```diff
@@ -56,3 +63,10 @@ export default ReleasesSelectControl;
 const StyledBadge = styled(Badge)`
   flex-shrink: 0;
 `;
+
+const ButtonLabelWrapper = styled('span')`
+  width: 100%;
+  text-align: left;
+  display: inline-grid;
+  grid-auto-flow: column;
```

Comment:
![Screen Shot 2022-07-15 at 10 01 50 AM](https://user-images.githubusercontent.com/22846452/179180142-c68b047e-fb86-4c11-89bd-ac299f7dbe21.png)

The chip is stretched out when this has more items selected. I think flexbox would work well here to achieve this styling

```suggestion
  display: flex;
  justify-content: space-between;
  align-items: center;
```